src/plugins_types/cert_exp_time.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * @file cert_exp_time.c
+ * @author Roman Janota <janota@cesnet.cz>
+ * @brief libnetconf2-netconf-server certificate-expiration-time type plugin.
+ *
+ * Copyright (c) 2024 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include "plugins_types.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+#include "libyang.h"
+
+#include "compat.h"
+#include "ly_common.h"
+#include "plugins_internal.h" /* LY_TYPE_*_STR */
+
+/**
+ * @page howtoDataLYB LYB Binary Format
+ * @subsection howtoDataLYBTypesCertExpTime certificate-expiration-time (libnetconf2-netconf-server)
+ *
+ * | Size (B) | Mandatory | Type | Meaning |
+ * | :------  | :-------: | :--: | :-----: |
+ * | string length | yes | `char *` | time offset relative to the current time |
```

Comment:
Update.

Reply:
Renamed `cert-exp-time` to `time-period`.